src/main/java/com/solvd/gui/pages/common/ProductsPageBase.java

```diff
@@ -19,11 +19,17 @@
 import java.util.Optional;
 
 public abstract class ProductsPageBase extends AbstractComponentSelectingPage {
-    ProductCategory productCategory;
+    @Getter
+    private String relativeUrl;
 
     @FindBy(css = ".products .product-items .product-item")
     private List<ProductCardBase> productCards;
 
+    // this element is used as load marker, and > selector is used in order to
+    // distinguish between search and products page
+    @FindBy(css = ".main > .products.wrapper")
```

Comment:
Could you please use more stable locator

Reply:
It seems pretty stable to me, it's used as load marker, so it also needs to be uniqe. Thats why I'm using child selector (`>`) to distinguis it from SearchProductPage, which is very similar. Also, there doesn't seems to be any better way to locate this element, since there are no ids nearby. Also the classes don't seem likely to change their names.

<img width="650" alt="Screenshot 2024-06-28 at 13 23 04" src="https://github.com/MKacprzakSolvd/carina-automation-learning/assets/170347259/7e6230a5-f61f-4ada-9bcd-8657f0ad8dd1">